### ISSUE TITLE
Docs: `search`: regex / operators clarification

### ISSUE THREAD

**Issue by joeflack4** [INCATools/ontology-access-kit#614]
## Overview
I was reading through https://incatools.github.io/ontology-access-kit/intro/tutorial01.html#search and also compared against https://incatools.github.io/ontology-access-kit/cli.html#runoak-search, and in the former, there are some special regex-like syntax that you can use, like `^`, and `~`, but that is not mentioned in the CLI command docs. Just wondering if: (i) it should be, (ii) if `^` and `~` is an exhaustive list, or if there are more.

**Comment by cmungall** [comment]
Good catch! The search syntax is powerful but the documentation is not well
organized
